src/components/PlayerBar/PlayerBar.js

```diff
@@ -37,29 +35,13 @@ export default function PlayerBar(props) {
         onVolumeChange,
         songEnded,
         currentTime,
-        onCueTimeChange
+        onCueTimeChange,
+        currentTimeFormated
     } = props;
 
-    // Set formated duration
-    useEffect(() => {
-        setFormatedDuration(getTime(duration));
-    }, [duration]);
-
-    // Set formated current time
-    useEffect(() => {
-        setFormatedCurrentTime(getTime(currentTime));
-    }, [currentTime]);
-
-    // Format time (eg. 140 to 2:20)
-    const getTime = dur => {
-        return (
-            Math.floor(dur / 60) + ':' + ('0' + Math.floor(dur % 60)).slice(-2)
-        );
-    };
-
```

Comment:
I see that you delete those and created a songUtil, at App.js { OnTimeUpdate } at the audio ref at later version of the code there is usage of parse.int that also can be refracted

Reply:
song duration & song current time will be in sec at redux store so I will keep the util file to convert the seconds in a string.